segfault_bug.py

```diff
@@ -0,0 +1,52 @@
+import subprocess
```

Comment:
Move to `scripts/experimental` and add docstring so maybe in the future we can mess with this again.